runtime/appruntime/metrics/prometheus/prometheus.go

```diff
@@ -155,6 +157,32 @@ func (x *Exporter) getMetricData(now time.Time, collected []metrics.CollectedMet
 	return data
 }
 
+func getSysMetrics(now time.Time) []*prompb.TimeSeries {
+	sysMetrics := system.ReadSysMetrics()
+	return []*prompb.TimeSeries{
+		{
+			Labels: []*prompb.Label{{
+				Name:  "__name__",
+				Value: "memory_usage_bytes",
+			}},
+			Samples: []*prompb.Sample{{
+				Value:     float64(sysMetrics.MemoryUsageBytes),
+				Timestamp: FromTime(now),
+			}},
+		},
+		{
+			Labels: []*prompb.Label{{
+				Name:  "__name__",
+				Value: "num_go_routines",
```

Comment:
Maybe to stick with the "<what>_<unit>" scheme this should be "goroutine_count" (also goroutine is one word, not two). But would be good to review all metrics we intend to add and make sure we have a consistent naming scheme for all of them.

Reply:
Since the other metric is called `e_requests_total`, then we should call this one `e_goroutines_total`, then?

Reply:
Now that I think about it, I avoided using the `total` suffix for the number of goroutines because that's a gauge. I think `count` would also suggest that being a counter?

Reply:
True. What does prometheus call it?

Reply:
The Prometheus Go client calls it `go_goroutines`.

Reply:
Ok. Do we have a list of the runtime metrics they track? I think we should look at the whole list of runtime metrics and come up with a consistent naming strategy instead of naming things one-by-one.